src/components/toast/toast.js

```diff
@@ -213,7 +213,9 @@ function MdToastProvider($$interimElementProvider) {
         var opts = {
           template: [
             '<md-toast md-theme="{{ toast.theme }}" ng-class="{\'md-capsule\': toast.capsule}">',
-              '<span flex>{{ toast.content }}</span>',
+              '<span flex role="alert" aria-relevant="all" aria-atomic="true">' +
```

Comment:
My testing showed that this worked with NVDA on Firefox and VoiceOver with Safari.